src/builder.cr

```diff
@@ -174,8 +174,24 @@ module Mint
       {runtime + compiled.to_s, type_checker.artifacts}
     end
 
+    def get_service_worker_utils(sw_utils_path)
```

Comment:
Won't `sw_utils_path` be the same as the `runtime_path`?

Reply:
Just tested this and the user provided runtime will be added to the service worker, but I'm not sure if it's an issue or not, the user provided runtime might have the `isMatchingRoute` function, but the custom runtime feature is not documented so :man_shrugging: 

It _feels like having a separate flag_ for the custom service worker runtime should be the way to go, but it's not necessary to implement it in this PR so you can remove that part.